bootstrapvz/plugins/debconf/__init__.py

```diff
@@ -1,6 +1,6 @@
 def validate_manifest(data, validator, error):
     from bootstrapvz.common.tools import log_check_call, rel_path
-    validator(data, rel_path(__file__, 'manifest-schema.yml'))
+    validator(data, rel_path(__file__, 'schema.yaml'))
```

Comment:
I think i was really tired when i did my commit... No idea why i did this again... anyways - Ahem - fixed